Tests/Functional/Bundle/TestBundle/Controller/SerializerErrorController.php

```diff
@@ -32,6 +32,7 @@ public function exceptionAction()
 
     /**
      * @View
+     * @group legacy
```

Comment:
and ``@group legacy`` should be added only on test covering your BC layers. This is not used in a test covering BC layers, so it should not be flagged as such (it would hide the fact that the testsuite will break on Symfony 3.0)